### PR TITLE
Replace requirements.txt with Pipfile and lock for use with Pipenv

### PR DESCRIPTION
**Pull request by jcox-dev**
Change to using Pipenv for dependencies and virtual environment management.

Setup: `pip install -U pip pipenv`
Run: `pipenv run scripts/run_app.sh`

Fixes: #2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM python:3.6
```
Comment (shahi645) [inline Dockerfile]: Should be 3.4.2 to keep inline with Survey Runner? Or is there a reason we need 3.6?

**Comment by shahi645** [comment]
@jcox-dev  
Should we integrate a README as part of this PR?

**Comment by jcox-dev** [comment]
@shahi645 it would be good to have but I think a separate PR for that would make more sense. Maybe open a new issue?

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 
+RUN pip install -U pipenv==8.2.7
```
Comment (jonnyshaw89) [inline Dockerfile]: I think we should remove -U as it contradicts the pinned version? 
Thats probably why codacy is moaning.

**Pull request merged**